view/frontend/web/js/view/payment/method-renderer/razorpay-method.js

```diff
@@ -173,6 +178,10 @@ define(
                         name: this.user.name,
                         contact: this.user.contact,
                         email: this.user.email
+                    },
+                    _: {
+                        integration: 'Magento',
+                        integration_version: data.maze_version,
```

Comment:
This will send magento version. We want to send razorpay-magento version.

Reply:
Actually it's probably useful to send both. Damn. Will discuss with @umanghome how we can do that.

Reply:
Added parent (framework) version as well.